From paul@uprocera.co.uk  Sun Feb  7 20:32:23 2010
From: Paul Elms <paul@uprocera.co.uk>
Date: Thu, 04 Feb 2010 08:24:28 +0000
Subject: Staging: comedi: fix brace coding style in comedi_fops.c
To: gregkh@suse.de, wfp5p@virginia.edu, shawn.bohrer@gmail.com, fmhess@users.sourceforge.net
Message-ID: <1265271868.2483.2.camel@arthur>

From: Paul Elms <paul@uprocera.co.uk>

This patch to the comedi_fops.c file fixes a brace warning found by the checkpatch.pl tool

Signed-off-by: Paul Elms <paul@uprocera.co.uk>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/comedi/comedi_fops.c |    6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

--- a/drivers/staging/comedi/comedi_fops.c
+++ b/drivers/staging/comedi/comedi_fops.c
@@ -2004,12 +2004,10 @@ void comedi_event(struct comedi_device *
 	if (async->cb_mask & s->async->events) {
 		if (comedi_get_subdevice_runflags(s) & SRF_USER) {
 			wake_up_interruptible(&async->wait_head);
-			if (s->subdev_flags & SDF_CMD_READ) {
+			if (s->subdev_flags & SDF_CMD_READ)
 				kill_fasync(&dev->async_queue, SIGIO, POLL_IN);
-			}
-			if (s->subdev_flags & SDF_CMD_WRITE) {
+			if (s->subdev_flags & SDF_CMD_WRITE)
 				kill_fasync(&dev->async_queue, SIGIO, POLL_OUT);
-			}
 		} else {
 			if (async->cb_func)
 				async->cb_func(s->async->events, async->cb_arg);
